docs/FAQ.md

```diff
@@ -37,6 +37,23 @@ See [revsets] and [templates] for further guidance.
 
 Use `jj log -r ..`. The `..` [operator] lists all visible commits in the repo, excluding the root (which is never interesting and is shared by all repos).
 
+### Can I monitor how `jj log` evolves?
+
+The simplest way to monitor how the history as shown by `jj log` evolves is by using the `watch(1)` command (or [hwatch](https://github.com/blacknon/hwatch) or [viddy](https://github.com/sachaos/viddy)).
+For example:
+
+```sh
+watch jj --ignore-working-copy log --color=always
+```
+
+This will continuously update the (colored) log output in the terminal.
+The `--ignore-working-copy` option avoids conflicts with manual operations during the creation of snapshots.
+Martin used this in a [tmux](https://github.com/tmux/tmux/wiki) pane during his presentation [Jujutsu - A Git-compatible VCS](https://www.youtube.com/watch?v=LV0JzI8IcCY).
+
```

Comment:
Optional (or I could add it): I'd add an example of a more complicated command.

> Here is an example of a more complicated `watch` invocation, using some Unix tools:
>
> ```shell
> watch --color -t -- "
>      jj --ignore-working-copy --color=always diff --stat | grep -Ev '^\d files'
>      jj --ignore-working-copy log --color=always"
> ```

Reply:
On second thought, maybe this is not such a great idea.

The problem is that `diff --stat` would only be updated after `jj` commands, which would be noticeable. So, we'd have to explain the tradeoff of keeping it that way, removing one of the `--ignore-working-copy`-ies, or using `core.watchman.register_snapshot_trigger=true`.

The downside of the last two is the increased chance of concurrency causing divergent change ids or stale workspaces. Also, using both of them makes the chance larger.